server/commands/battle.py

```diff
@@ -856,6 +868,146 @@ def send_info_guild(client):
+    """
+    This command will let you create an item
+    Examples: /create_item steak hp x1.5, /create_item rottenmeat hp +-20
+    Usage: /create_item <ItemName> <stat> <operation>
+    """
+    args = shlex.split(arg)
+    if len(args) < 3:
+        client.send_ooc("Argument error\n: /create_item <stat> <operation>")
+        return
+    if args[1].lower() not in ["hp", "mana", "atk", "def", "spa", "spd", "spe"]:
+        client.send_ooc("Argument error\n: /create_item <ItemName> <stat> <operation>")
+        return
+    if args[2][0] not in ["x", "+"]:
+        client.send_ooc("Argument error\n: /create_item <ItemName> <stat> <operation>")
+        return
+    try:
+        num = float(args[2][1:])
+    except:
+        client.send_ooc("Argument error\n: /create_item <ItemName> <stat> <operation>")
+        return
+    if args[2][0] == "x" and float(args[2][1:]) < 0:
+        client.send_ooc("you cannot multiply for a negative number")
+        return
+    client.area.battle_items[f"{args[0]}"] = {}
+    client.area.battle_items[f"{args[0]}"]["stat"] = args[1].lower()
+    client.area.battle_items[f"{args[0]}"]["operation"] = args[2][0]
+    client.area.battle_items[f"{args[0]}"]["number"] = num
+    client.send_ooc(f"{args[0]} has been created!")
+
+
+@mod_only(hub_owners=True)
+def ooc_cmd_give_item(client, arg):
+    """
+    This command will give an item to a fighter
+    Usage: /give_item <TargetID> <ItemName>
+    """
+    args = shlex.split(arg)
+    if len(args) < 2:
+        client.send_ooc("Argument error\n: /give_item <TargetID> <item>")
+        return
+    if not args[0].isdigit():
+        client.send_ooc("Argument error\n: /give_item <TargetID> <item>")
+        return
+    try:
+        target = client.server.client_manager.get_targets(
+            client, TargetType.ID, int(args[0]), False
+        )[0]
+    except:
+        client.send_ooc("Argument error\n: /give_item <TargetID> <item>")
+        return
+    if target not in client.area.clients:
+        client.send_ooc("Target is not in the current area!")
+        return
+    if args[1] not in client.area.battle_items:
+        client.send_ooc("Item doesn't exist!")
+        return
+    if target.battle is None:
+        client.send_ooc("Target isn't using a fighter in this moment!")
+        return
+    target.battle.bag.append(args[1])
+    client.send_ooc(f"{args[1]} has been given to {target.battle.fighter}")
+    send_stats_fighter(target)
+    target.send_ooc(f"You have obtained {args[1]}")
+
+
+def ooc_cmd_use_item(client, arg):
+    """
+    This command will let you use an item from your bag
+    Usage: /use_item <ItemName>
+    """
+    if client.battle is None:
+        client.send_ooc("You are not using fighter in this moment!")
+        return
+    if arg not in client.battle.bag:
+        client.send_ooc("Item not found in your bag")
+        return
+    item = client.area.battle_items[arg]
+    if item["stat"] == "hp":
```

Comment:
why can an item can only modify a single stat?

What about items that trade your HP for Mana? Increase attack but reduce defense?